### ISSUE TITLE
Couldn't load 'Argon2PasswordHasher' algorithm library: No module named 'argon2'

### ISSUE THREAD

**Issue by oscmedgon** [WeblateOrg/docker#1597]
### Describe the issue

I have Weblate installed some time ago via helm chart in my K8s cluster, today I upgraded the helm chart and it fails to run due to a missing module, I tried to workaround by installing it in the image via a custom command but there are some missing dependencies, this could be related to this problem that people are experiencing with Django [error](https://stackoverflow.com/questions/46416985/django-issue-with-argon2-hasher-in-live-production)

### I already tried

- [X] I've read and searched [the documentation](https://docs.weblate.org/).
- [X] I've searched for similar issues in this repository.

### Steps to reproduce the behavior

1. Upgrade the helm chart
2. Go to the weblate created pod
3. See error

### Expected behavior

Expect Weblate to run without errors.

### Screenshots

![Screenshot from 2022-09-07 11-28-09](https://user-images.githubusercontent.com/29759668/188844019-7e190bfd-69d0-4f65-bbde-f08ba172e9c9.png)

### Exception traceback

```pytb
402 static files copied to '/app/data/static'.
Compressing... done
Compressed 6 block(s) from 230 template(s) for 0 context(s).
Updating user admin
[2022-09-07 09:24:45,586: WARNING/167] Handled exception: ValueError: Couldn't load 'Argon2PasswordHasher' algorithm library: No module named 'argon2'
Traceback (most recent call last):
  File "/usr/local/lib/python3.7/dist-packages/django/contrib/auth/hashers.py", line 183, in _load_library
    module = importlib.import_module(mod_path)
  File "/usr/lib/python3.7/importlib/__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1006, in _gcd_import
  File "<frozen importlib._bootstrap>", line 983, in _find_and_load
  File "<frozen importlib._bootstrap>", line 965, in _find_and_load_unlocked
ModuleNotFoundError: No module named 'argon2'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/local/bin/weblate", line 8, in <module>
    sys.exit(main())
  File "/usr/local/lib/python3.7/dist-packages/weblate/runner.py", line 34, in main
    utility.execute()
  File "/usr/local/lib/python3.7/dist-packages/django/core/management/__init__.py", line 395, in execute
    self.fetch_command(subcommand).run_from_argv(self.argv)
  File "/usr/local/lib/python3.7/dist-packages/django/core/management/base.py", line 330, in run_from_argv
    self.execute(*args, **cmd_options)
  File "/usr/local/lib/python3.7/dist-packages/weblate/utils/management/base.py", line 42, in execute
    return super().execute(*args, **options)
  File "/usr/local/lib/python3.7/dist-packages/django/core/management/base.py", line 371, in execute
    output = self.handle(*args, **options)
  File "/usr/local/lib/python3.7/dist-packages/weblate/auth/management/commands/createadmin.py", line 93, in handle
    if password is not None and not user.check_password(password):
  File "/usr/local/lib/python3.7/dist-packages/django/contrib/auth/base_user.py", line 112, in check_password
    return check_password(raw_password, self.password, setter)
  File "/usr/local/lib/python3.7/dist-packages/django/contrib/auth/hashers.py", line 50, in check_password
    is_correct = hasher.verify(password, encoded)
  File "/usr/local/lib/python3.7/dist-packages/django/contrib/auth/hashers.py", line 322, in verify
    argon2 = self._load_library()
```

### Additional context

Versions
```
k8s: v1.22.12
nginx-ingress-controller: v1.3.1
weblate-chart: v0.4.13
weblate: v4.5.2-1 (weblate/weblate:4.5.2-1)
```

**Comment by nijel** [comment]
The module was included in the 4.6-1 tag (via 598e5c6349441ab8be1e059974e3cab8af3ceccc), and Argon was also enabled in Weblate 4.6 (via https://github.com/WeblateOrg/weblate/commit/a1e9098253b). Maybe you should upgrade :-). Or remove your customization to enable Argon2.

**Comment by oscmedgon** [comment]
I've been tinkering a bit on the problem. The problem, as you pointed out, is for older docker images, but the latest helm chart installs the Weblate version v4.5.2-1. What I did was to update the deployment with an updated docker image and resolve the incompatibilities in the configuration, I also needed to add the CSRF_TRUSTED_ORIGINS in the Weblate config map and now it works.
I close the issue, thank you for your support.
